python/lsst/ctrl/oods/msgIngester.py

```diff
@@ -96,20 +97,30 @@ async def ingest(self, butler_file_list):
         # for each butler, attempt to ingest the requested file,
         # Success or failure is noted in a message description which
         # will send out via a CSC logevent.
-        LOGGER.info("ingest called")
+        LOGGER.info("calling ingest")
         try:
             await self.butler.ingest(butler_file_list)
         except Exception as e:
             LOGGER.warning("Exception: %s", e)
+        LOGGER.info("ingest completed")
 
-    def _helper_done_callback(self, task):
-        LOGGER.info("called")
+    def _queue_helper_done_callback(self, task):
+        self._helper_done_callback(task, "dequeue task completed")
+
+    def _helper_done_callback(self, task, msg=None):
+        # LOGGER.info("called")
         if task.exception():
             try:
                 task.result()
             except Exception as e:
                 LOGGER.info(f"Task {task}: {e}")
-        LOGGER.info("completed")
+        if msg is not None:
+            LOGGER.info(msg)
+        else:
+            # LOGGER.info("completed")
+            pass
```

Comment:
I might be missing some context, but if no action is required when `msg` is `None`, I'd remove the else-branch entirely.